### DISCUSSION TITLE
Why are Addy e-mail addresses disallowed at `mastodon.social`?

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [mastodon/mastodon#27431] [Q&A] [answered]
#### Reproduction

1.	Visit [`mastodon.social/auth/edit`](https://mastodon.social/auth/edit).

1.	Enter `*@Addy.io` into `user_email`:

	> uses a disallowed e-mail provider

**Comment by roke-julian-lockhart** [comment]
I have requested that this be remediated at [`issues/27436`](https://github.com/mastodon/mastodon/issues/27436#issue-1945559401).

**Comment by nightpool** [comment, marked as answer]
addy.io is one of many services that people can use to generate disposable, anonymous email addresses. These services are overwhelmingly used for spam. blocking these email providers is unfortunately necessary to maintain a spam-free environment.

**Reply by roke-julian-lockhart** [reply]
> https://github.com/mastodon/mastodon/discussions/27431#discussioncomment-7294569

@nightpool, the same applies to Outlook. I can create 5 addresses per account, and creating an account at https://signup.live.com/signup is as simple as creating an account at https://app.addy.io/register.

**Reply by satonotdead** [reply]
> addy.io is one of many services that people can use to generate disposable, anonymous email addresses. These services are overwhelmingly used for spam. blocking these email providers is unfortunately necessary to maintain a spam-free environment.

Sorry but [that's not true](https://github.com/anonaddy/anonaddy/issues/466#issuecomment-1569718168). I'm staying away from Mastodon for things like this.

**Reply by tiritto** [reply]
Disappointing.

**Reply by satonotdead** [reply]
> addy.io is one of many services that people can use to generate disposable, anonymous email addresses. These services are overwhelmingly used for spam. blocking these email providers is unfortunately necessary to maintain a spam-free environment.

@nightpool May I know the source of this information? Because I understand Addy.io as an open-source privacy e-mail alias service.

**Reply by somethingstarted** [reply]
well, if you wanted us to use mastadon you should change your mind and allow addy.io. come up with other anti-spam measures instead. it's not that hard.

**Comment by jakejoh** [comment]
I am using addy.io with my own domain, this is the first service that rejects my address.

**Reply by roke-julian-lockhart** [reply]
> https://github.com/mastodon/mastodon/discussions/27431#discussioncomment-8782590

@jakejoh, https://github.com/anonaddy/anonaddy/discussions/467#discussioncomment-8786218 may be of use. However, note that, per https://github.com/anonaddy/anonaddy/discussions/484#discussion-5416354, this isn't the sole service to do so (although it's very rare).
